_layouts/default.html

```diff
@@ -43,7 +43,15 @@
           {% if site.aux_links != nil %}
             <ul class="list-style-none text-small aux-nav">
               {% for link in site.aux_links %}
-                <li class="d-inline-block my-0{% unless forloop.last %} mr-2{% endunless %}"><a href="{{ link.last }}">{{ link.first }}</a></li>
+                <li class="d-inline-block my-0{% unless forloop.last %} mr-2{% endunless %}">
+                  {% if site.aux_links_new_tab %}
+                  <a href="{{ link.last }}" target="_blank" rel="noopener noreferrer">
+                  {% else %}
+                  <a href="{{ link.last }}">
+                  {% endif %}
```

Comment:
You could write it like this:

```html
<a href="{{ link.last }}"{% if site.aux_links_new_tab != false %} target="_blank" rel="noopener noreferrer"{% endif %}>
```

Reply:
Yeah, I think that's a bit cleaner. Updated the branch!